schema/wasi-args.capnp

```diff
@@ -0,0 +1,21 @@
+@0xb47a0b67171659f7;
+
+using import "wasi-common.capnp".Errno;
+using import "wasi-common.capnp".Size;
+
+interface Args {
+  # Read command-line argument data.
+  # The sizes of the buffers should match that returned by argsSizesGet().
+  get @0 (
+    argvBuf :List(Text) # A pointer to a buffer to write the argument string data.
+  ) -> (
+    error :Errno
+  );
+
+  # Return command-line argument data sizes.
+  sizesGet @1 () -> (
```

Comment:
assuming i can use these files as some sort of data to generate information i can use in my library or language or whatever, shouldn't these functions be snake_case?

Reply:
so there are a couple of routes to go here. Capn proto schema style should be camelcase, but then the doc and interface generators can could spit out snake_case. Also we can add annotations to generate the current naming if there are other discrepancies.